File: Backend/Solution/Application.Repository/Models/Shopping.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Application.Repositories.Models
{
    public partial class Shopping
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public double? Amount { get; set; }
        public string IdentificatorOfMoney { get; set; }

        public virtual User User { get; set; }
    }
}

File: Frontend/app-quote-shoppings-final/src/app/quote-shopping/models/Shopping.ts
export class Shopping 
{
   userid:number ;
   amount:number ;
   identificatorOfMoney:string;
   constructor(userid:number,amount:number,identificatorOfMoney:string)
   { 
       this.userid =userid;
       this.amount =amount;
       this.identificatorOfMoney=identificatorOfMoney;
   }
}